src/components/MainMenu.jsx

```diff
@@ -1,18 +1,87 @@
 import styled from "styled-components";
 import { Routes, Route, NavLink, Link } from "react-router-dom";
+import logo from "../images/Logo_nacHHaltig.png";
+import NavLinks from "./NavLinks";
+
+import Restaurants from "../pages/Restaurants";
+import Shopping from "../pages/Shopping";
+import Beauty from "../pages/Beauty";
+import Favourites from "../pages/Favourites";
+import Calculator from "../pages/Calculator";
+import NotFound from "../pages/NotFound";
 
 function MainMenu() {
```

Comment:
Is it really the main menu or rather the main page/index page?

Reply:
I think it's both, the MainMenu and also the main page, I will rename it :)